lib/css/components/notices.less

```
@@ -1,252 +1,267 @@
+            --_no-btn-bg-focus: var(--yellow-500);
+
+            .dark-mode({
+                --_no-bg: var(--yellow-600);
+                --_no-fc: var(--white);
+            });
+            .highcontrast-mode({
+                --_no-bg: var(--yellow-700);
+                --_no-fc: var(--white);
+            });
+        }
     }
-    &__warning:not(&__important) {
-        .highcontrast-mode({
-            --_no-bg: var(--yellow-200);
-        });
+
+    // CHILD ELEMENTS
+    code {
+        background: var(--_no-code-bg, transparent);
     }
-    &__warning&__important {
-        --_no-bc: var(--_no-bg);
-        --_no-bg: var(--yellow-400);
-        --_no-btn-focus-bg: var(--yellow-500);
-        --_no-btn-active-bg: var(--yellow-600);
-        --_no-fc: var(--black-900);
+    & &--btn {
+        &:focus,
+        &:hover {
+            background: var(--_no-btn-bg-focus);
+        }
+        &:active {
+            background: var(--_no-btn-bg-active);
+        }
 
-        .dark-mode({
-            --_no-bg: var(--yellow-600);
-            --_no-fc: var(--white);
-        });
-        .highcontrast-mode({
-            --_no-bg: var(--yellow-700);
-            --_no-fc: var(--white);
-        });
+        color: inherit;
+        padding: var(--su8);
     }
+
+    // STYLES MODIFIED BY COMPONENT-SPECIFIC CUSTOM PROPERTIES
+    background: var(--_no-bg);
+    border-color: var(--_no-bc);
+    color: var(--_no-fc);
+
+    border-style: solid;
+    font-size: var(--fs-body1);
 }
 
 // Banner
 // TODO deprecate .s-banner (by turning it into a modifier on .s-notice)
 // This would reduce the amount of CSS we ship to the client and simplify our codebase
 .s-banner {
-    .construct-notice-component();
-    --_no-x-offset: 0;
-    //      When we use .s-banner, we need to adjust the padding-top on
-    //      the body tag. This class correctly adjusts the body padding ONLY if
-    //      the notice is one line. If it wraps to multiple lines, more classes or
-    //      (ideally) JS will need to be used to determine the notice's height
-    //      at the time of render. The padding value is determined like so:
-    //
-    //      50px (top bar) + 44px (notice height) - 1px (bottom border)
-    //
-    //      The borders subtraction are necessary to neatly tuck everything together.
-    //  ----------------------------------------------------------------------------
-    &__body-pt {
-        padding-top: 93px;
-    }
-    //  If you want to put the banner above the topbar
-    &.is-pinned {
-        z-index: calc(var(--zi-navigation-fixed) + 1);
-    }
-    // If you want to hide and reveal the banner
-    &[aria-hidden="true"] {
-        --_no-x-offset: -50px; // TODO convert to su vars
-        visibility: hidden;
+    --_no-x-offset: 0; // [1]
+    .construct-notice-component(s-banner);
+
+    &[aria-hidden="true"] { // If you want to hide and reveal the banner
+        --_no-x-offset: calc(var(--su48) + var(--su2) * -1) // -50px
```

You missed a semicolon, @dancormier.

```suggestion
        --_no-x-offset: calc(var(--su48) + var(--su2) * -1); // -50px
```